mmedit/models/mattors/gca.py

```diff
@@ -91,8 +91,9 @@ def forward_test(self,
             dict: Contains the predicted alpha and evaluation result.
         """
         pred_alpha = self._forward(torch.cat((merged, trimap), 1))
-
-        pred_alpha = pred_alpha.cpu().numpy().squeeze()
+        if torch.onnx.is_in_onnx_export():
```

Comment:
Since there is already a `forward_dummy()`, maybe you can make use of that?

Reply:
OK, I have submitted a new commit which use forward_dummy for onnx exporting